server/gql/_shared.graphql

```diff
@@ -0,0 +1,108 @@
+scalar Upload
+scalar Any
+scalar DateTime
+scalar URL
+scalar Lang
+scalar FileSize
+scalar TranslatedString
+scalar Cursor
+
+# Meta Type
+
+interface Node {
+  id: ID!
+}
+
+enum NodeType {
+  ASSET
+  USER
+  TEAM
+  PROJECT
+  PLUGIN
+  SCENE
+  PROPERTY_SCHEMA
+  PROPERTY
+  DATASET_SCHEMA
+  DATASET
+  LAYER_GROUP
+  LAYER_ITEM
+}
+
+# Pagination
+
+type PageInfo {
+  startCursor: Cursor
+  endCursor: Cursor
+  hasNextPage: Boolean!
+  hasPreviousPage: Boolean!
+}
+
+input Pagination{
+  first: Int
+  last: Int
+  after: Cursor
+  before: Cursor
+}
+
+# Basic types
+
+type LatLng {
+  lat: Float!
+  lng: Float!
+}
+
+type LatLngHeight {
+  lat: Float!
+  lng: Float!
+  height: Float!
+}
+
+type Camera {
+  lat: Float!
+  lng: Float!
+  altitude: Float!
+  heading: Float!
+  pitch: Float!
+  roll: Float!
+  fov: Float!
+}
+
+type Rect {
+  west: Float!
+  south: Float!
+  east: Float!
+  north: Float!
+}
+
+type Typography {
+  fontFamily: String
+  fontWeight: String
+  fontSize: Int
+  color: String
+  textAlign: TextAlign
+  bold: Boolean
+  italic: Boolean
+  underline: Boolean
+}
+
+enum TextAlign {
+  LEFT
+  CENTER
+  RIGHT
+  JUSTIFY
+  JUSTIFY_ALL
+}
```

Comment:
@rot1024 
It seems those are not used types, should I remove them?

Reply:
No, they are used